Tasks/GithubRelease/operations/Release.ts

```diff
@@ -0,0 +1,144 @@
+import tl = require("vsts-task-lib/task");
+import util = require("util");
+import path = require("path");
+import fs = require('fs');
+import mime = require('mime');
+import { Utility, Inputs } from "./Utility";
+import { WebRequest, sendRequest, WebResponse } from "./webClient";
+
+export class Release {
+
+    public static async createRelease(repositoryName: string, releaseTitle: string, isDraft: boolean, isPrerelease: boolean): Promise<WebResponse> {
+        const tag = tl.getInput(Inputs.tagCreate) || undefined;
```

Comment:
Should we pass tags and target as an argument to the method?

Reply:
Yes. will take in next PR. have some pending work related to this